setup.py

```diff
@@ -74,7 +74,7 @@ def get_version():
     license='BSD License',
     packages=['markdown', 'markdown.extensions'],
     python_requires='>=3.6',
-    install_requires=["importlib-metadata>='4.4';python_version<'3.10'"],
+    install_requires=["importlib-metadata>=4.4;python_version<'3.10'"],
```

Comment:
I'm wondering if `python_version<'3.10'` needs to be unquoted as well.

Reply:
Never mind, it seems documented examples use quotes, so it should be fine: https://www.python.org/dev/peps/pep-0508/#environment-markers. I actually don't see examples without.